### ISSUE TITLE
String conversion assumes ascii string encoding (was: UnicodeDecodeError)

### ISSUE THREAD

**Issue by hxhc** [stuart-cls/python-opusfc-dist#9]
I am trying to convert a opus file into csv, but it raises such an error. Doesn't know how to fix it.
I tried different environments (shown below), but it raised the same error.

OS: win10, Chinese / Linux mint English
Python: 3.6.5 anaconda
OpusFC: 1.2.0 / 1.10

```
     10     dbs = opus.listContents(f)
---> 11     data = opus.getOpusData(f, dbs[0])
     12     spectrum = np.array([data.y])
     13     spectrum = spectrum.T


opusFC.pyx in opusFC.getOpusData (opusFC.c:11826)()

opusFC.pyx in opusFC.getOpusData.getParams (opusFC.c:7677)()

UnicodeDecodeError: 'ascii' codec can't decode byte 0xb2 in position 3: ordinal not in range(128)
````

you can get a sample file [here](https://zjueducn-my.sharepoint.com/:u:/g/personal/zsheng_zju_edu_cn/EVBegYEbRW5MnTJ7G5bPwWcBqemlbx5yAHbQpMudw3Nq1g?e=azv9No)

Thank you in advance.

**Comment by stuart-cls** [comment]
Thank you for your report. The module currently assumes all strings are encoded in 'ascii'.

This can be fixed in the next release but I can't commit to a date for that at this time.

**Comment by hxhc** [comment]
Thank you, looking forward to  the next release.

**Comment by aurelianjh** [comment]
I have the same issue with a set of OPUS files. @stuart-cls, by any chance did you get to work on the next release?

> OPUS_ref_raw = op.getOpusData(str(ipath), dbs[idbs]) # Load data of interest from OPUS file
> Traceback (most recent call last):
> 
>   File "<ipython-input-39-bdceecbf94a6>", line 1, in <module>
>     OPUS_ref_raw = op.getOpusData(str(ipath), dbs[idbs]) # Load data of interest from OPUS file
> 
>   File "opusFC.pyx", line 762, in opusFC.getOpusData
> 
>   File "opusFC.pyx", line 858, in opusFC.getOpusData
> 
>   File "opusFC.pyx", line 609, in opusFC.getOpusData.getParams
> 
> UnicodeDecodeError: 'ascii' codec can't decode byte 0x97 in position 31: ordinal not in range(128)

**Comment by kidpixo** [comment]
Hi, 
I also got the same error. I cannot understand how to get a copy of the code to try to fix on my own and give back to the community...

**Comment by stuart-cls** [comment]
Fixed in 1.2.1
